src/DGtal/shapes/MeshHelpers.h

```diff
@@ -0,0 +1,149 @@
+/**
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ **/
+
+#pragma once
+
+/**
+ * @file MeshHelpers.h
+ * @author Jacques-Olivier Lachaud (\c jacques-olivier.lachaud@univ-savoie.fr )
+ * Laboratory of Mathematics (CNRS, UMR 5127), University of Savoie, France
+ *
+ * @date 2017/02/11
+ *
+ * Header file for module MeshHelpers.cpp
+ *
+ * This file is part of the DGtal library.
+ */
+
+#if defined(MeshHelpers_RECURSES)
+#error Recursive header files inclusion detected in MeshHelpers.h
+#else // defined(MeshHelpers_RECURSES)
+/** Prevents recursive inclusion of headers. */
+#define MeshHelpers_RECURSES
+
+#if !defined MeshHelpers_h
+/** Prevents repeated inclusion of headers. */
+#define MeshHelpers_h
+
+//////////////////////////////////////////////////////////////////////////////
+// Inclusions
+#include <iostream>
+#include "DGtal/base/Common.h"
+#include "DGtal/topology/CCellEmbedder.h"
+#include "DGtal/topology/CDigitalSurfaceContainer.h"
+#include "DGtal/topology/DigitalSurface.h"
+#include "DGtal/shapes/TriangulatedSurface.h"
+#include "DGtal/shapes/Mesh.h"
+//////////////////////////////////////////////////////////////////////////////
+
+namespace DGtal
+{
+
+  /////////////////////////////////////////////////////////////////////////////
+  // template class MeshHelpers
+  /**
+   * Description of template class 'MeshHelpers' <p>
+   * \brief Aim: Static class that provides builder and converters between meshes.
+   */
+  class MeshHelpers
+  {
+    // ----------------------- Static services ------------------------------
+  public:
+    /// Builds a triangulated surface (class TriangulatedSurface) from
+    /// a mesh (class Mesh). Note that a triangulated surface contains
+    /// only triangles, so faces of the input mesh are (naively)
+    /// triangulated (triangles (0,1,2), (0,2,3), (0,3,4),
```

Comment:
(e.g. a planar face with vertices {0,1,2,..} is triangulated with triangles (0,1,2), (0,2,3), (0,3,4)...)